tests/testsuite/build_script.rs

```diff
@@ -4030,20 +4270,10 @@ fn warnings_printed_on_vv() {
 
     p.cargo("build -vv")
         .with_stderr_data(str![[r#"
-[UPDATING] `dummy-registry` index
-[LOCKING] 2 packages to latest compatible versions
-[DOWNLOADING] crates ...
-[DOWNLOADED] bar v0.1.0 (registry `dummy-registry`)
-[COMPILING] bar v0.1.0
-[RUNNING] `[..] rustc --crate-name build_script_build [..]`
-[RUNNING] `[..] [ROOT]/foo/target/debug/build/bar-[HASH]/build-script-build`
+...
 [WARNING] bar@0.1.0: foo
 [WARNING] bar@0.1.0: bar
-[RUNNING] `[..] rustc --crate-name bar [..]`
-[COMPILING] foo v0.5.0 ([ROOT]/foo)
-[RUNNING] `[..] rustc --crate-name foo [..]`
-[FINISHED] `dev` profile [unoptimized + debuginfo] target(s) in [ELAPSED]s
-
+...
 "#]])
```

Comment:
There a reason this test was changed?  Using `...` makes things not update as cleanly

Reply:
Reverted to the old snapshot.